### DISCUSSION TITLE
Ability for topics to be in lists

### DISCUSSION THREAD

**Discussion by MattIPv4** [community/community#8235] [Other Feature Feedback, Questions, & Ideas]
Topics can be starred by users on GitHub, much like repositories. The (new?) [stars view](https://github.com/stars/MattIPv4) clearly shows both topics and repositories as "top-level" subjects.

However, the dropdown to add something to a list on the star button only appears on repositories, and not on topics.

Would be great if topics could be categorised in the same way as repos.

**Comment by brianlovin** [comment]
Agreed! In the future, we'd love to see more things eligible to be "listed" — topics, gists, profiles, etc. I'll add your +1 to our tracking issue 😊
